### PR TITLE
Adds tests and makes SQS integrations deleteable

### PR DESCRIPTION
**Pull request by jeremybmerrill**
Hi @GabeIsman,

This PR fixes an outstanding issue with my previous one where SQS integrations couldn't be deleted (because I thickheadedly failed to change a reference to `@slack_integration` to `@sqs_integration`. 

I have also included the tests from before, but I have no preference as to whether you merge those or not. If you'd prefer a pull request with ONLY the deletion fix, I'd be happy to do that too, just lmk

### CODE REVIEW

**Comment by tommeagher** [comment]
Awesome. Thank you, @jeremybmerrill. We'll take a look soon. We'll holler if we have any questions.

File: spec/controllers/sqs_integrations_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    {
+      queue_url: "https://sqs.us-east-1.amazonaws.com/1234567890/fake-klaxon-sqs"
+    }
+  }
+
+  let(:invalid_attributes) {
+    {
+      queue_url: "http://sqs.us-east-1.amazonaws.com/1234567890/fake-klaxon-sqs" # note not HTTP
```
Comment (GabeIsman) [inline spec/controllers/sqs_integrations_controller_spec.rb]: s/HTTP/HTTPS/

File: spec/controllers/sqs_integrations_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        post :create, {:sqs_integration => invalid_attributes}, valid_session
+        expect(response).to render_template("new")
+      end
+    end
+  end
+
+  describe "PUT #update" do
+    context "with valid params" do
+      let(:new_attributes) {
+        skip("Add a hash of attributes valid for your model")
```
Comment (GabeIsman) [inline spec/controllers/sqs_integrations_controller_spec.rb]: Fill in attributes

File: spec/controllers/sqs_integrations_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      }
+
+      it "updates the requested sqs_integration" do
+        sqs_integration = SqsIntegration.create! valid_attributes
+        put :update, {:id => sqs_integration.to_param, :sqs_integration => new_attributes}, valid_session
+        sqs_integration.reload
+        skip("Add assertions for updated state")
```
Comment (GabeIsman) [inline spec/controllers/sqs_integrations_controller_spec.rb]: Missing assertion

File: spec/requests/sqs_integrations_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "SqsIntegrations", type: :request do
+  describe "GET /sqs_integrations" do
+    it "works! (now write some real specs)" do
```
Comment (GabeIsman) [inline spec/requests/sqs_integrations_spec.rb]: Update this description :), no more real specs needed

**Comment by jeremybmerrill** [comment]
Cool thanks!

**Pull request merged**